test/controllers/deregister/ConfirmStopBeingPsaControllerSpec.scala

```diff
@@ -41,10 +40,12 @@ class ConfirmStopBeingPsaControllerSpec extends ControllerSpecBase with ScalaFut
 
   import ConfirmStopBeingPsaControllerSpec._
 
+//  implicit val hc: HeaderCarrier = HeaderCarrier()
+
   "ConfirmStopBeingPsaController" must {
 
     "return to session expired if psaName is not present" in {
-      val result = controller().onPageLoad()(fakeRequest)
+      val result = controller()(hc).onPageLoad()(fakeRequest)
```

Comment:
Do we need to explicitly pass header carrier? Is it defined as an implicit value `ControllerSpecBase`?

Reply:
It is not defined as an implicit value in `ControllerSpecBase` as it would be used in any controller that extends `ControllerSpecBase` whether it is needed or not.